### PR TITLE
Add/home campaign chip

### PR DESCRIPTION
**Pull request by LanesGood**
Adds campaign chip to homepage campaign card

This required updating the topstats file with a db join. @kamicut I'm not sure if this has too much information in the db connection string now, mind taking a look?

image

### CODE REVIEW

**Comment by LanesGood** [comment]
@kamicut as discussed, we'll review and merge this in once tests are working after merging #647.

**Review by kamicut — APPROVED** [review]

**Pull request merged**
